aineko/templates/first_aineko_pipeline/cookiecutter.json

```diff
@@ -0,0 +1,9 @@
+{
+    "project_name": "My Awesome Pipeline",
+    "project_slug": "{{ cookiecutter.project_name.lower()|replace(' ', '_')|replace('-', '_')|replace('.', '_')|trim() }}",
+    "project_description": "Behold my awesome pipeline!",
+    "authors": "John Doe <johndoe@gmail.com>",
+    "pipeline_slug": "test-aineko-pipeline",
+    "_aineko_core_directory": "/Users/lionellloh/Projects/convex-labs/aineko-core",
```

Comment:
Do we still need this and the validation logic? Aineko is now available on PyPi.

Reply:
Good catch, we don't now